.github/workflows/balena.yml

```diff
@@ -32,8 +32,9 @@ env:
     export-database \
     import-database \
     e2e-server-previous-dump
-
-  LIVECHAT_PORT: 443
+  INTEGRATION_JELLYFISH_APP_ID: jellyfish
+  INTEGRATION_JELLYFISH_APP_SECRET: changeme
+  HYDRA_ADMIN_HOST: http://hydra:4445
```

Comment:
Maybe we could also drop this line? It seems to be the same value as the default defined here: https://github.com/product-os/jellyfish-environment/blob/master/lib/variables/hydra/defaults.ts#L1